src/IConfig.ts

```diff
@@ -82,6 +82,10 @@ export interface IConfig {
 
     provisioning?: {
         enable: boolean;
-        auth_callbck: string;
+        require_public_room: boolean;
```

Comment:
`require_public_room` is not marked required in `config/slack-config-schema.yaml`